pinot-broker/src/main/java/org/apache/pinot/broker/routing/BrokerRoutingManager.java

```diff
@@ -603,6 +603,18 @@ public boolean routingExists(String tableNameWithType) {
     return _routingEntryMap.containsKey(tableNameWithType);
   }
 
+  /**
+   * Returns whether the given table is enabled
+   * @param tableNameWithType Table name with type
+   * @return Whether the given table is enabled
+   */
+  public boolean isTableEnabled(String tableNameWithType) {
+    if (_routingEntryMap.containsKey(tableNameWithType)) {
```

Comment:
We can save one map lookup by directly calling `get()` and check whether the return value is `null`